runtime/doc/lua.txt

```diff
@@ -4433,4 +4433,53 @@ tohtml.tohtml({winid}, {opt})                         *tohtml.tohtml.tohtml()*
         (`string[]`)
 
 
+==============================================================================
+Lua module: vim.net                                                  *vim.net*
+
+vim.net.download({url}, {opts})                           *vim.net.download()*
+    Asynchronously download a file
+
+    Parameters: ~
+      • {url}   (`string`) Request URL
+      • {opts}  (`table?`) Additional options
+
+                Example: >lua
+                    -- Download a file
+                    -- The file will be saved in the `cwd` with the name `anything`
+                    vim.net.download("https://httpbingo.org/anything")
+
+                    -- Download a file to a path
+                    -- The file will be saved in `/tmp/somefile`
+                    vim.net.download("https://httpbingo.org/anything", {
+                      as = "tmp/somefile",
+                    })
```

Comment:
Only reading this help, I am not sure if this is a typo or an intended behavior: is setting `as = "tmp/somefile"` indeed result into saving in '/tmp/somefile'? Because the first one doesn't have leading `/` which means it is a relative path and should result into saving as 'somefile' file in the 'tmp' subdirectory of current working directory.

Reply:
Nice catch, it was a typo